app/main.py

```diff
@@ -1,6 +1,16 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def wrapper(*args) -> Any:
+        key = ", ".join(map(str, [*args]))
+        cache_value = cache_dict.get(key)
+        if cache_value is not None:
+            print("Getting from cache")
+            return cache_value
```

Comment:
You could shorten this code with the walrus operator:
```suggestion
        if cache_value := cache_dict.get(key):
            print("Getting from cache")
            return cache_value
```